modules/std.lobster

```diff
@@ -119,21 +119,30 @@ def qsort(xs, lt):
 def qsort_in_place(xs, lt):
     def rec(s, e):
         let l = e - s
-        if l > 1:
-            let pivot = xs[s]
-            var sp = s + 1
-            var ep = e
-            while sp < ep:
-                let c = xs[sp]
-                if lt(c, pivot):
-                    xs[sp - 1] = xs[sp]
-                    sp++
-                else:
-                    xs[sp] = xs[--ep]
-                    xs[ep] = c
-            xs[--sp] = pivot
-            rec(s, sp)
-            rec(ep, e)
+
+        switch l:
+            case 0, 1:
```

Comment:
why this over `if l <= 1:` ?

Reply:
Right, this PR is a draft to demonstrate a crash, this part isn't actually a serious "I think we should actually change it" yet; the crash is the only thing here, and not even really "for review" as this is a draft PR ;-)